data/mods/Magiclysm/items/enchanted_cloaks.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "id": "Wildshape Cloak: Fish",
+    "type": "ARMOR",
+    "name": "Wildshape Cloak: Fish",
+    "description": "A magical cloak that shimmers like fresh water under a warm sun. It can be worn to morph your body to a form that is excellent for swimming, but ill-suited for combat.",
+    "looks_like": "cloak_leather",
+    "weight": "2060 g",
+    "volume": "3050 ml",
+    "price": 100000,
+    "symbol": "w",
+    "color": "blue",
+    "covers": [ "torso", "arm_l", "arm_r", "leg_l", "leg_r" ],
+    "coverage": 0,
+    "encumbrance": 0,
+    "msg": "You feel your body change its form!",
+    "flags": [ "OUTER", "OVERSIZE", "NONCONDUCTIVE", "WATER_FRIENDLY", "RAINPROOF" ],
+    "relic_data": { "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "mutations": [ "GILLS", "MEMBRANE", "AMPHIBIAN" ] } ] }

```

Comment:
i would suggest using enchantment ids for the passive effects rather than defining them explicitly in the item, because that way if they need to be updated later the ingame items will be updated alongside with them.
Basically you define an enchantment as a separate object, and then do 
```
"passive_effects": [ { "id": "enchantment_id" } ]
```

Reply:
the fish wildshape also is apparently missing the WILDSHAPE:FISH mutation

Reply:
I made all of the changes you suggested - although it still is failing the Lint check and I cannot figure out why. I ran everything through the web lining tool and it seems correct. Do you have any suggestions? Thanks!